website/cue/reference/components/sinks/gcp_chronicle_unstructured.cue

```diff
@@ -68,58 +69,7 @@ components: sinks: gcp_chronicle_unstructured: {
 		notices: []
 	}
 
-	configuration: {
-		api_key: configuration._gcp_api_key
-		credentials_path: {
-			category:    "Auth"
-			common:      true
-			description: "The filename for a Google Cloud service account credentials JSON file used to authenticate access to the Cloud Storage API. If this is unset, Vector checks the `GOOGLE_APPLICATION_CREDENTIALS` environment variable for a filename.\n\nIf no filename is named, Vector will attempt to fetch an instance service account for the compute instance the program is running on. If Vector is not running on a GCE instance, you must define a credentials file as above."
-			required:    false
-			type: string: {
-				default: null
-				examples: ["/path/to/credentials.json"]
-			}
-		}
-		endpoint: {
-			common:        false
-			description:   "The endpoint to send data to."
-			relevant_when: "region is not set"
```

Comment:
💭 just thinking of this now even though I know some of my components probably had this.
Are we not going to have this concept going forward since the auto generated docs don't have it?